### PR TITLE
Consider Expanding the Allowed Characters for HTS Ticker Symbols

### PR DESCRIPTION
**Pull request by scalemaildev**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-0000-template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2021-11-19
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/231
+---
+
+## Abstract
+
+Allow some symbols and non-capitalized alpha characters to be used in HTS ticker symbols.
```
Comment (sergmetelin) [inline HIP/hip-0000-template.md:15]: Have you tried creating symbols with symbols? 
both `name` and `symbol` can be any Unicode sequence at all
the only restriction is that the UTF-8 encoding of this Unicode cannot contain the 0 bytes (`NUL`)

Comment (scalemaildev) [reply HIP/hip-0000-template.md:15]: I have not but when reading through the SDK documentation [here](https://docs.hedera.com/guides/docs/sdks/tokens/define-a-token) it states:

```
The publicly visible token symbol. It is UTF-8 capitalized alphabetical string identifying the token. The token symbol is not unique. Maximum of 100 characters.
```

Do you know if this is still true? I can do some tests later myself to find out.

Comment (sergmetelin) [reply HIP/hip-0000-template.md:15]: https://docs.hedera.com/guides/docs/sdks/tokens/define-a-token - we've updated documentation now to clarify!

Comment (sergmetelin) [reply HIP/hip-0000-template.md:15]: If this addresses the functionality in question in this HIP - feel free to close the PR!

**Pull request closed without merging**